crates/astria-gossipnet/src/network.rs

```diff
@@ -131,43 +136,119 @@ impl Network {
 
         // build a gossipsub network behaviour
         let gossipsub = gossipsub::Behaviour::new(
-            gossipsub::MessageAuthenticity::Signed(local_key),
+            gossipsub::MessageAuthenticity::Signed(keypair),
             gossipsub_config,
         )
         .map_err(|e| eyre!("failed to create gossipsub behaviour: {}", e))?;
```

Comment:
use modern rust formatting (I also like to add backticks to make variable things stand out in errors or in general formatted logs):
```rust
        .map_err(|e| eyre!("failed to create gossipsub behaviour: `{e}`"))?;
```